hummingbot/market/liquid/liquid_market.pyx

```diff
@@ -0,0 +1,1346 @@
+        return retval
+
+    async def _update_order_status(self):
+        """
+        Pulls the rest API for for latest order statuses and update local order statuses.
+
+        Example updated dict:
+        {
+            'id': 1817834299,
+            'order_type': 'market',
+            'quantity': '10.0',
+            'disc_quantity': '0.0',
+            'iceberg_total_quantity': '0.0',
+            'side': 'buy',
+            'filled_quantity': '10.0',
+            'price': '0.0004998',
+            'created_at': 1575261189,
+            'updated_at': 1575261189,
+            'status': 'filled',
+            'leverage_level': 1,
+            'source_exchange': None,
+            'product_id': 500,
+            'margin_type': None,
+            'take_profit': None,
+            'stop_loss': None,
+            'trading_type': 'spot',
+            'product_code': 'CASH',
+            'funding_currency': 'ETH',
+            'crypto_account_id': None,
+            'currency_pair_code': 'CELETH',
+            'average_price': '0.0004998',
+            'target': 'spot',
+            'order_fee': '0.000005',
+            'source_action': 'manual',
+            'unwound_trade_id': None,
+            'trade_id': None,
+            'client_order_id': 'buy-CELETH-1575261189133282',
+            'settings': None,
+            'trailing_stop_type': None,
+            'trailing_stop_value': None,
+            'executions': [{
+                'id': 232722006,
+                'quantity': '10.0',
+                'price': '0.0004998',
+                'taker_side': 'buy',
+                'created_at': 1575261189,
+                'my_side': 'buy'
+            }],
+            'stop_triggered_time': None
+        }
+        """
+        cdef:
+            double current_timestamp = self._current_timestamp
+
+        if current_timestamp - self._last_order_update_timestamp <= Constants.UPDATE_ORDERS_INTERVAL:
+            return
+
+        tracked_orders = list(self._in_flight_orders.values())
+        listed_orders = await self.list_orders()
+        order_dict = dict((
+            str(listed_order["id"]), listed_order)
+            for listed_order in listed_orders.get("models", []))
+
+        for tracked_order in tracked_orders:
+            exchange_order_id = await tracked_order.get_exchange_order_id()
+            order_update = order_dict.get(str(exchange_order_id))
+            if order_update is None:
+                self.logger().network(
+                    f"Error fetching status update for the order {tracked_order.client_order_id}: "
+                    f"{order_update}.",
+                    app_warning_msg=f"Could not fetch updates for the order {tracked_order.client_order_id}. "
+                                    f"Check API key and network connection."
+                )
+                continue
+
+            order_status = order_update.get("status")
```

Comment:
Do look into the schema definitions for order statuses. I have already dropped an inquiry regarding this, but for now the string enums for `status` are `[live,canceled,filled]`